src/qmapshack/poi/CRawPoi.cpp

```diff
@@ -74,8 +74,87 @@ poi_t CRawPoi::toPoi() const
     poi_t poi;
     poi.pos = coordinates;
     poi.name = getName();
-    poi.desc = data.join("<br>\n");
     poi.icon = garminIcon;
+
+    const QRegularExpression emailRegex ("(contact:email)=(.+)", QRegularExpression::UseUnicodePropertiesOption);
```

Comment:
Imho this should be ´static const` as you do not want to initialize the regex multiple times